src/main/java/app/model/agents/Grid.java

```diff
@@ -84,4 +85,22 @@ public double getCellSize()
     {
         return cellSize;
     }
+
+    public int getCellsExplored()
+    {
+        // Loop through grid and divide explored cells by total cells
+        int cellsExplored = 0;
+        for(Cell[] row : grid)
```

Comment:
At the moment the grid is not dynamically sized, but rather a single large map of 1000 x 1000 cells. So the coverage may be skewed by the large size